### ISSUE TITLE
JSON output: Issuer & Subject improvement

### ISSUE THREAD

**Issue by botastic** [nabla-c0d3/sslyze#404]
Hey,

I'm using SSLyze within my own application and parse SSLyze's JSON output to import the results, because I'm not using Python.

Recently I stumbled upon an error in my parser, which I think is rather a bad design choice in the generation of the JSON output. Here's a partial string of the 'Subject' field within the certificate: 

`[...], organizationName=RICOH COMPANY,LTD., [...]`

The way my parsing works is the following:
1. split at every `,`
2. split at every `=`
3. evaluate the substrings

You might already see my Problem the comma in `COMPANY,LTD.` results in a split and the substring `LTD.` can not be further processed.

I'm well aware that I could add a whitespace as a prefix to the comma in the first step, but there's no guarantee that this pattern won't be in any of the values.

The way I see it there are a couple of possible solutions:
1. Replace the separator in [this line](https://github.com/nabla-c0d3/sslyze/blob/c77ed6396fefac26d19865842e9d5bc847f431be/sslyze/plugins/utils/certificate_utils.py#L63) with some other separator (maybe a newline character). Unfortunately this could also fail at some point.
2. Simply return the values as a JSON array with a field for every oid.

I'd guess that there is a standard/definition/best practice how these OIDs are formed and what characters are valid. But if everybody would conform to those we'd probably not need a security scanner at all ;)

Because of the more robust design ( and of course the simpler parsing on my part :D ) I'd strongly prefere the second solution.

Best regards

**Comment by nabla-c0d3** [comment]
In the next release the output will be like this:
```
"subject": {
    "attributes": [
        {
            "oid": {
                "dotted_string": "2.5.4.6",
                "name": "countryName"
            },
            "rfc4514_string": "C=US",
            "value": "US"
        },
        {
            "oid": {
                "dotted_string": "2.5.4.10",
                "name": "organizationName"
            },
            "rfc4514_string": "O=GeoTrust Inc.",
            "value": "GeoTrust Inc."
        },
        {
            "oid": {
                "dotted_string": "2.5.4.3",
                "name": "commonName"
            },
            "rfc4514_string": "CN=GeoTrust Primary Certification Authority",
            "value": "GeoTrust Primary Certification Authority"
        }
    ],
    "parsing_error": null,
    "rfc4514_string": "C=US,O=GeoTrust Inc.,CN=GeoTrust Primary Certification Authority"
},
```

**Comment by botastic** [comment]
That looks great!

Thank you very much for your work :)

**Comment by nabla-c0d3** [comment]
Fixed in v3.0.0.
